tests/sentry/snuba/metrics/test_extraction.py

```diff
@@ -166,24 +166,18 @@ class TestCreatesOndemandMetricSpec:
             ("percentile(transaction.duration, 1)", "transaction.duration>0"),
             ("count_if(transaction.duration,equals,0)", "transaction.duration:>0"),
             ("count_if(transaction.duration,notEquals,0)", "transaction.duration:>0"),
-            (
-                "count()",
-                "project:a-1 route.action:CloseBatch level:info",  # custom tags not supported by standard metrics
```

Comment:
I'm only removing `level:info` since it not anymore a supported on-demand query.